tests/testthat/test-paginate_listing.R

```diff
@@ -218,38 +218,59 @@ testthat::test_that("pagination repeats keycols in other pages", {
 
   # Simplified test
   mf_pages <- as_listing(tibble("a" = rep("1", 25), "b" = seq(25)), key_cols = "a") %>%
-    paginate_to_mpfs(lpp = 10)
+    paginate_to_mpfs(lpp = 12)
 
   testthat::expect_snapshot(cat(toString(mf_pages[[3]])))
 
   # Warning from empty key col
   mf_pages <- suppressWarnings(
     testthat::expect_warning(
       as_listing(tibble("a" = rep("", 25), "b" = seq(25)), key_cols = "a") %>%
-        paginate_to_mpfs(lpp = 10)
+        paginate_to_mpfs(lpp = 12)
     )
   )
   mf_pages <- suppressWarnings(
     as_listing(tibble("a" = rep("", 25), "b" = seq(25)), key_cols = "a") %>%
-      paginate_to_mpfs(lpp = 10)
+      paginate_to_mpfs(lpp = 12)
   )
 
   testthat::expect_snapshot(
     cat(toString(mf_pages[[3]]))
   )
 })
 
+
+testthat::test_that("pagination repeats keycols in other pages (longer test)", {
```

Comment:
This test is already in place in {formatters}. I am not sure but it could be already present also a version here. I believe it is in the form of a regression test. I need to double-check

Reply:
our previous test was not sufficient, that's why this bug was not picked up earlier. this is a cheap operation, but help us to make sure we don't miss these

Reply:
I see! I did not see the fix. Thanks for this